tribe-ext-main-blog-loop/index.php

```diff
@@ -0,0 +1,95 @@
+<?php
+/**
+ * Plugin Name:     The Events Calendar Extension: Main Blog Loop
+ * Description:     Additional options for including events in the main blog loop.
+ * Version:         1.0.0
+ * Extension Class: Tribe__Extension__Main_Blog_Loop
+ * Author:          Modern Tribe, Inc.
+ * Author URI:      http://m.tri.be/1971
+ * License:         GPLv2 or later
+ * License URI:     https://www.gnu.org/licenses/gpl-2.0.html
+ */
+
+// Do not load directly.
+if ( ! defined( 'ABSPATH' ) ) { die( '-1' ); }
+// Do not load unless Tribe Common is fully loaded.
+if ( ! class_exists( 'Tribe__Extension' ) ) { return; }
```

Comment:
If you want to do one-liners, use ternaries.
E.g.:
```php
// Don't load directly
defined( 'WPINC' ) or die;
```

Otherwise I think it would be better to split it into multiple lines.